bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LocalBookieEnsemblePlacementPolicy.java

```diff
@@ -88,14 +89,16 @@ public BookieSocketAddress replaceBookie(int ensembleSize, int writeQuorumSize,
     }
 
     @Override
-    public List<Integer> reorderReadSequence(ArrayList<BookieSocketAddress> ensemble, List<Integer> writeSet,
-                                             Map<BookieSocketAddress, Long> bookieFailureHistory) {
+    public IntArrayList reorderReadSequence(
```

Comment:
We are breaking compatibility. We already did break compatibility for policy in 4.5....
Anyway if you have I would like to have a BK  class

Reply:
I am fine with this, if it helps with performance - because the stability of this interface is `Evolving`. so it is okay to break the compatibility in minor release: https://github.com/apache/bookkeeper/blob/master/bookkeeper-common/src/main/java/org/apache/bookkeeper/common/annotation/InterfaceStability.java#L41

https://github.com/apache/bookkeeper/blob/master/bookkeeper-server/src/main/java/org/apache/bookkeeper/client/EnsemblePlacementPolicy.java#L195